clang/include/clang/Basic/BuiltinTemplates.td

```diff
@@ -0,0 +1,23 @@
+//===--- BuiltinTemplates.td - Clang builtin template aliases ---*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+class BuiltinTemplate<string prototype> {
+  string Prototype = prototype;
+}
+
+def __make_integer_seq : BuiltinTemplate<
+  "template <template <class T, T... Ints> class IntSeq, class T, T N>">;
```

Comment:
Its unfortunate that we have this much "just arbitrary text" here.  I don't have a better idea, and no real wish to try to make it 'better', but this makes me grumpy and I wanted to let you know.

ALSO, and something that IS actionable: Can we document the importance of the defined 'name'?  Since it is the actual name of the builtin that we're exposing to the user?  Typically these (names in .td files) are internal names that are generally arbitrary, so it would be nice to document that:

"An entry here will introduce a template builtin with the name X, and whatever-else-we-are-doing-here."

Reply:
Do you think it would be better to give the templates an internal name to be more consistent?